dashboard/src/components/AppView/SecretsTable/SecretsTable.test.tsx

```diff
@@ -64,3 +91,23 @@ it("renders a table with several secrets", () => {
       .key(),
   ).toBe(secret2);
 });
+
+it("renders a secret table with a secret and an error", () => {
```

Comment:
Either implement ,delete, or mark this test case as pending (using `xit`)

Reply:
oops, I missed this